pkg/util/kernel/fs.go

```diff
@@ -3,23 +3,82 @@
 package kernel
 
 import (
+	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/moby/sys/mountinfo"
 )
 
+// MountInfoPidPath returns the path to the mountinfo file of a pid in /proc
+func MountInfoPidPath(pid int32) string {
+	return filepath.Join(util.HostProc(), fmt.Sprintf("/%d/mountinfo", pid))
```

Comment:
Couldn't this be wrong if the host proc is mapped at `/host/proc`? If the container is in a different PID namespace, the PID returned by `os.Getpid()` will not be correct in `/host/proc`, only correct in `/proc`?